app/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.microsoft.office365.msgraphsnippetapp">
 
     <application
         android:name="com.microsoft.office365.msgraphsnippetapp.application.SnippetApp"
-        android:allowBackup="true"
+        android:allowBackup="false"
+        android:fullBackupContent="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:replace="android:allowBackup">
```

Comment:
This is related to the following issue in ADAL for Android.
AzureAD/azure-activedirectory-library-for-android/issues/472